customization/mail-templates/enquiry-notification-consultant.html

```diff
@@ -39,7 +39,7 @@ <h1 style='color: #FFFFFF; font-size: 30px; font-weight: bold; line-height: 40px
                      <tr>
                         <td cellspacing='0' cellpadding='0' style="padding: 0;" colspan="2">
                            <div style="margin: 20px 20px 0; font-size: 16px; line-height: 21px;">
-                              <b style="font-size: 20px; line-height: 28px; color: #333;">Liebe_r <span
+                              <b style="font-size: 20px; line-height: 28px; color: #333;"><span data-th-text="#{mail.label.dear}"/> <span
                                     data-th-text="${name}"></span>,</b>
                               <br /><br />
                               in der Beratungsstelle <span data-th-text="${beratungsstelle}"></span> ist eine neue
```

Comment:
Are we not translating the whole template?

Reply:
fixed